### PR TITLE
Hardware sensors

### PR DESCRIPTION
**Pull request by Oblarg**
Write section for sensor hardware

### CODE REVIEW

File: source/docs/hardware/sensors/analog-inputs-hardware.rst [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+
+TODO: add picture
+
+.. |RoboRIO Analog Inputs| image:: images/analog-inputs-hardware/roborio-analog-inputs.png
+.. |ground| unicode:: 0x23DA
+
+Footnotes
+---------
+
+.. [1] A 12-bit resolution yields :math:`2^12`, or 4096 different values.  For a 5V range, that's an effective resolution of approximately 1.2 mV, or .0012V.  The actual accuracy specification is plus-or-minus 50mV, so the discretization is not the limiting factor in the measurement accuracy.
```
Comment (Daltz333) [inline source/docs/hardware/sensors/analog-inputs-hardware.rst]:
```
:math:`2^12`
```
should be
```
:math:`2^{12}`
```

**Comment by Daltz333** [comment]
@kevin-o-connor Are the various sensors in this Pull Request okay for merge?

**Comment by Kevin-OConnor** [comment]
> @Kevin-OConnor Are the various sensors in this Pull Request okay for merge?

The gyro article should reference the Analog Devices gyros and IMUs that have been available the past few years (the 3 listed here: https://www.analog.com/en/landing-pages/001/first.html). Other than that, seems fine to me

**Comment by Kevin-OConnor** [comment]
@Oblarg I added the Analog Devices stuff to the gyro and accel articles and made one other tweak. There are clearly still pictures to add, but is this good to go from your perspective?

**Comment by Oblarg** [comment]
Looks great to me!

**Comment by Kevin-OConnor** [comment]
> 
> 
> Looks great to me!

Okay. I'm going to open a separate PR with these changes rebased onto develop (giving you credit in the message) and then come back and close this one.

**Comment by Kevin-OConnor** [comment]
Merged as #97

**Pull request closed without merging**